README.md

```diff
@@ -1,2 +1,233 @@
-# identity-sdks-dotnet
-SDKs related to the identity written using .NET technologies.
+# Asgardio .NET OIDC SDK
+
+[![Build Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fwso2.org%2Fjenkins%2Fjob%2Fasgardio%2Fjob%2Fasgardio-dotnet-oidc-sdk%2F&style=flat)](https://wso2.org/jenkins/job/asgardio/job/asgardio-dotnet-oidc-sdk/) [![Stackoverflow](https://img.shields.io/badge/Ask%20for%20help%20on-Stackoverflow-orange)](https://stackoverflow.com/questions/tagged/wso2is)
+[![Join the chat at https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE](https://img.shields.io/badge/Join%20us%20on-Slack-%23e01563.svg)](https://join.slack.com/t/wso2is/shared_invite/enQtNzk0MTI1OTg5NjM1LTllODZiMTYzMmY0YzljYjdhZGExZWVkZDUxOWVjZDJkZGIzNTE1NDllYWFhM2MyOGFjMDlkYzJjODJhOWQ4YjE)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/wso2/product-is/blob/master/LICENSE)
+[![Twitter](https://img.shields.io/twitter/follow/wso2.svg?style=social&label=Follow)](https://twitter.com/intent/follow?screen_name=wso2)
+---
+
+Asgardio .NET OIDC SDK enables you to add OIDC based login, logout to your .NET apps in a  simple manner.
+
+- [Getting Started](#getting-started)
+- [How it works](#how-it-works)
+- [Integrating OIDC SDK to your existing .NET application](#integrating-oidc-sdk-to-your-existing-net-application)
+- [Building from the source](#building-from-the-source)
+- [Contributing](#contributing)
+  * [Reporting issues](#reporting-issues)
+- [License](#license)
+
+## Getting started
+You can experience the capabilities of Asgardio .NET OIDC SDK by following this small guide which contains main sections listed below.
```

Comment:
TYPO: *as listed below

Reply:
Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/4/commits/3945c817476dfe36be1ba0eac68d777f7d6dd006